### PR TITLE
Fix jest testing to work in other timeZones

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: none
affects: none
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 91% | 91.07% | -0.07%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  You have included an invalid app. Valid values are: Student, Teacher, Parent, None
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**React Native** | 92.3% | 92.3% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Student** | 93.12% | 93.12% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: dangerfile.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ function handleAffects (message) {
     app[0].toUpperCase() + app.slice(1).toLowerCase()
   )
   const valid = ['Student', 'Teacher', 'Parent', 'None']
-  const invalid = apps.filter(app => valid.includes(app))
+  const invalid = apps.filter(app => !valid.includes(app))
```
Comment (armstrongnate) [inline dangerfile.js]: Sneaky! This slipped right passed me.

**Pull request merged**